ansible_collections/arista/avd/roles/eos_designs/python_modules/custom_structured_configuration/avdstructuredconfig.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
+
+from functools import cached_property
+
+from ansible_collections.arista.avd.plugins.filter.natural_sort import natural_sort
+from ansible_collections.arista.avd.plugins.plugin_utils.avdfacts import AvdFacts
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import get
+
+
+class AvdStructuredConfig(
+    AvdFacts,
+):
+    """
+    The AvdStructuredConfig Class is imported used "yaml_templates_to_facts" to render parts of the structured config.
+
+    "yaml_templates_to_facts" imports, instantiates and run the .render() method on the class.
+
+    The Class uses AvdFacts, as the base class, to get the render, keys and other attributes.
```

Comment:
render is actually coming from this class

Reply:
updated